style.css

```diff
@@ -159,3 +159,323 @@
   right: 0;
   top: 87.09%;
 }
+
+.work {
+  background-color: rgb(229 229 229);
```

Comment:
- kindly make sure you change the background color as it was required by  the previews reviewer [here](https://github.com/Uchenna-ibute/my-portfolio/pull/3/files#r1007955572) It will make your design stick to the Figma design :+1:
use the one provided below
`background: #FFFFFF;`